src/app/pages/property-details/reviews/reviews.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { Router } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { APPCONFIG } from 'src/app/config';
+
+
+
+@Component({
+  selector: 'app-reviews',
+  templateUrl: './reviews.component.html',
+  styleUrls: ['./reviews.component.scss']
+})
+export class ReviewsComponent implements OnInit {
+  @Input() clientId: number;
+  subscription: Subscription = new Subscription();
+  id: number;
+  count: number;
+  review: string;
+  reviewer: any;
+  createdAt: string;
+  replies: Array<any>[];
```

Comment:
Awesome work @dorothyas, is there any reason you included replies because it has its separate task and I can't see where you used it.

Reply:
@Niyitangasam thank you for pointing that out. I have removed all the unnecessary variables